src/rtsp_player_controller.cc

```diff
@@ -489,8 +489,10 @@ void RTSPPlayerController::calculateAudioLevel(AVFrame* input_frame, AVSampleFor
 
 		for (int i = 0; i < nb_samples; i++) {
 			sample = (char)buff16[i];
-			sample = sample / 128.0; //Normalized to (+/- 1.0)
-			sum += (sample * sample);
+			//sample = sample / 128.0; //Normalized to (+/- 1.0)
+			//sum += (sample * sample);			
+			sample = (sample < 0)? -sample : sample; //abs
```

Comment:
I noticed there is also a function in `math.h`--`fabs` which returns absolute value of a floating-point number. This way is fine too.